astarte-message-hub/src/data/astarte_handler.rs

```diff
@@ -191,41 +194,42 @@ impl AstarteRunner for AstarteHandler {
     /// function.
     #[allow(dead_code)]
     async fn run(&mut self) {
-        use crate::proto_message_hub::AstarteMessage;
+        use astarte_message_hub_proto::proto_message_hub::AstarteMessage;
+
+        let Ok(astarte_data_event) = self.device_sdk.handle_events().await else {
+            warn!("Received empty event");
```

Comment:
Logging the possible error returned by `handle_events` would provide more diagnostic information. 
Consider making this a `match`.